src/main/java/org/mamute/model/Attachment.java

```diff
@@ -0,0 +1,77 @@
+package org.mamute.model;
+
+import br.com.caelum.vraptor.observer.upload.UploadedFile;
+import org.hibernate.annotations.Type;
+import org.joda.time.DateTime;
+import org.mamute.providers.SessionFactoryCreator;
+
+import javax.persistence.*;
+
+@Entity
+public class Attachment {
+
+	@GeneratedValue
+	@Id
+	private Long id;
+
+	@Transient
+	private UploadedFile file;
+
+	@Type(type = SessionFactoryCreator.JODA_TIME_TYPE)
+	private final DateTime createdAt = new DateTime();
+
+	@ManyToOne
+	private User owner;
+
+	private String ip;
```

Comment:
I decided to persist it for security reasons. With this file upload feature someone could attack the application by simply uploading a lot of files to fill the whole disk. By storing the ip address we can develop some kind of rate limiting of downloads by ip in the future.